### ISSUE TITLE
Failing APNS for Independent WatchOS6 app

### ISSUE THREAD

**Issue by RichAppz** [appfeel/node-pushnotifications#117]
Hey guys, I am not sure if this is of fault of the library, Apple, or something I have done wrong.

So basically the new Independent WatchOS6 apps can now have a push token of their own, don't need the phone. I am creating a token and passing to server no issue.

The issue I am getting is EVERY time I try and send the push I get Error: DeviceTokenNotForTopic I have tried with p8 - and key/cert pem - Both are passing the same errors.

Is there something missing from your end?

**Comment by vitalyster** [comment]
```
When using certificate-based authentication, the topic is usually your app's bundle ID. However, the topic may also correspond to the bundle ID of an Apple Watch complication or a background VoIP service. Look in the (1.2.840.113635.100.6.3.6) extension of your certificate to identify the possible topics for your push notifications
``

**Comment by RichAppz** [comment]
I am currently using the bundle ID of the app. Not using a complication or VoIP just standard independent WatchOS6 app.

**Comment by alex-friedl** [comment]
Should be fixed with #119
